src/main/java/org/opentripplanner/graph_builder/module/GraphCoherencyCheckerModule.java

```diff
@@ -1,77 +0,0 @@
-package org.opentripplanner.graph_builder.module;
-
-import org.opentripplanner.graph_builder.DataImportIssueStore;
-import org.opentripplanner.graph_builder.services.GraphBuilderModule;
-import org.opentripplanner.routing.graph.Edge;
-import org.opentripplanner.routing.graph.Graph;
-import org.opentripplanner.routing.graph.Vertex;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-
-/**
- * Check the every vertex and edge in the graph to make sure the edge lists and from/to
- * members are coherent, and that there are no edgeless vertices. Primarily intended for debugging.
- */
-public class GraphCoherencyCheckerModule implements GraphBuilderModule {
```

Comment:
I think this would be good to wire back again, since I've gotten burned a couple of times with this when working with island pruning

Reply:
Ok, I will take it out of the PR.